scripts/api/setup-optional-harvard.sh

```diff
@@ -55,4 +55,6 @@ curl http://localhost:8080/api/admin/datasetfield/load -X POST --data-binary @da
 curl http://localhost:8080/api/admin/datasetfield/load -X POST --data-binary @data/metadatablocks/customCHIA.tsv -H "Content-type: text/tab-separated-values"
 curl http://localhost:8080/api/admin/datasetfield/load -X POST --data-binary @data/metadatablocks/customDigaai.tsv -H "Content-type: text/tab-separated-values"
 curl http://localhost:8080/api/admin/datasetfield/load -X POST --data-binary @data/metadatablocks/custom_hbgdki.tsv -H "Content-type: text/tab-separated-values"
+# setup/update local solr instance with custom metadata block fields
+sudo -u solr ../../conf/solr/7.3.1/updateSchemaMDB.sh -t /usr/local/solr/server/solr/collection1/conf
```

Comment:
I'm not sure about adding this line to the script - it assumes that Solr is running locally; AND has a hard-coded path to the installation directory, AND the username under which solr is running... too many assumptions? 
I would maybe replace it with just an "echo: ATTENTION! you have installed optional metadata blocks, you now have to add the extra fields to the solr schema, by running the updateSchemaMDB.sh script, like this: ... etc. etc."

This may not be super important - I don't think we distribute this script publicly; it's not part of the installer bundle, and we (here at Harvard) may be the only installation actually running it... But that said, Solr is running on a different server, both in our production environment, and on some of our test instances... so the line above isn't going to work.

Reply:
@landreev good point. In aebc850da I switched it to an echo instead of actually running the script.